src/modules/article.js

```diff
@@ -5,8 +5,8 @@ const getCurrentArticle = async id => {
     const response = await axios.get(`/articles/${id}`);
     return response.data.article;
   } catch (error) {
-    if (error === 'Network Error') {
-      return { error: error.message }
+    if (error === "Network Error") {
+      return { error: error.message };
```

Comment:
how can you call `error.message` if error equals to "Network Error"?

Reply:
fixed